lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -0,0 +1,411 @@
+        else:
+            n0 = constant8 * ((triple_pt / cell) - 1.)
+            n1 = constant9 * np.log10(triple_pt / cell)
+            n2 = constant10 * (1. - (cell / triple_pt))
+            log_es = n0 - n1 + n2 + constant11
+            cell[...] = (np.power(10., log_es) * 100)
+    correction = 1. + 1E-8 * pressure * (4.5 + 6E-4 * data ** 2)
+    svp = temp.copy(data=data)
+    svp = svp * correction
+
+    # Tidy Up cube
+    svp.units = Unit('Pa')
+    svp.rename("saturated_vapour_pressure")
+    return svp
+
+
+def saturation_vapour_pressure_simple(temperature):
+    """
+    A simple saturation vapour pressure equation, considered to be within
+    1% of correct value for the given range.
+
+    Args:
+        temperature : iris.cube.Cube
+            Cube of temperature which will be converted to Kelvin
+            prior to calculation
+            Valid from -100C to 200
+
+    Returns:
+        results : iris.cube.Cube
+        Cube containing the saturation vapour pressure of the
+        air in Pa
+
+    References:
+        Numerical data and functional relationships in science and technology.
+        New series. Group V. Volume 4. Meteorology. Subvolume b. Physical and
+        chemical properties of the air, P36.
+    """
+    temp = temperature.copy()
+    temp.convert_units('K')
+    check_range(temp, 173., 373.)
+    data = temp.data
+    for cell in np.nditer(data, op_flags=['readwrite']):
+        if cell < 0:
+            cell[...] = np.exp(-5372.305844 / cell + 21.47824488)
+        else:
+            cell[...] = np.exp(-6147.795166 / cell + 24.31720686)
+    result = temp.copy(data=data)
+    result.units = Unit('hPa')
+    result.convert_units('Pa')
+    result.rename("saturated_vapour_pressure")
+    return result
+
+
+def humidity_ratio_fm_rh(temperature, relative_humidity, pressure):
+    ''' Function to compute humidity ratio given temperature pressure and
+    relative humidity.
+
+    Args:
+        temperature : iris.cube.Cube
+            Cube of temperature which will be converted to celsius
+            prior to calculation.
+        rel_humidity : iris.cube.Cube
+            Cube of relative humidity.
+        pressure : iris.cube.Cube
+            Cube of pressure which will be converted to kilopPascals
+            prior to calculation.
+
+    Returns
+        hr : iris.cube.Cube
+            humidity ratio cube with units (1).
+
+    References:
+        ASHRAE Fundamentals handbook (2005) Equation 22, 24, p6.8
+    '''
+    # Decouple local variables from variables supplied
+    # Ensure that variables are in correct units
+    temp = temperature.copy()
+    temp.convert_units('celsius')
+    rh = relative_humidity.copy()
+    rh.convert_units(1)
+    press = pressure.copy()
+    press.convert_units('kPa')
+    svp = saturation_vapour_pressure_goff_gratch(temp, press)
+    svp.convert_units('kPa')
+
+    # Calculation
+    result_numer = (0.62198 * rh.data * svp.data)
```

Comment:
Include a comment describing the calculation of partial pressure of water vapour (Pw), e.g. RH = Pw/Psat

Reply:
I thought this was the basis of this calculation. Am I wrong? If so, please include a comment that describes whatever this calculation is based upon so it's more obvious.